service/tests/playlist.test.ts

```diff
@@ -0,0 +1,40 @@
+import { MockMongoose } from "mock-mongoose";
+import mongoose, { Mongoose } from "mongoose";
+import CreateMongoDBConnection from "../../database/mongodb";
+import PlaylistService from "../playlist";
+
+describe("Test Suite - Playlist Service", () => {
+
+    const service = new PlaylistService();
+
+    beforeAll(async () => {
+        await CreateMongoDBConnection({
+            database: "test-nodejs",
+            port: "27017",
+            host: "localhost"
+        })
+    });
+
+    afterAll(async () => {
+        mongoose.disconnect();
+    });
+
+    it("test - add new song to playlist", async () => {
```

Comment:
You have to compare expected value with actual value. This test case is only calling a function.